### DISCUSSION TITLE
Demoparty

### DISCUSSION THREAD

**Discussion by dtinth** [creatorsgarten/discussions#2] [Event ideas]
ไม่แน่ใจว่าในเมืองไทยมีคนที่ชอบเกี่ยวกับ Procedural art หรือ Demoscene เยอะขนาดไหน อยากจัดงานที่คนสายนี้ได้มาปล่อยของโชว์ของกัน

Do you like to create **procedural art** with code? We would like to run an event that brings together people who are interested in **GLSL, shader art, generative visuals,** and **live coding.**

![Shader code generating showing Thai style pattern](https://user-images.githubusercontent.com/193136/186988734-f2b1949c-0bee-4503-bea5-c28d0d3a47c3.jpeg)

In the event, we might make a nice generative art using live coding tools like [Bonzomatic](https://github.com/Gargaj/Bonzomatic), [Hydra](https://hydra.ojack.xyz/), or [GLSL Sandbox](https://glslsandbox.com/), possibly synchronized to music. Then we can share the results, knowledge, and techniques with each other. Or we might play games like [_Shader Showdown_](https://www.facebook.com/shadershowdown) where people play head-to-head against each other to [create a shader within 30 minutes.](https://youtu.be/gmMPvUwyMxA?t=123)

Idea suggested by @heypoom.

**Comment by narze** [comment]
Also music? https://sonic-pi.net
